### DISCUSSION TITLE
Route::livewire('whatever', 'directory.component') throws "Unable to find component"

### DISCUSSION THREAD

**Discussion by johncarter-** [livewire/livewire#1055] [4. General]
I can't seem to render a component from a sub-directory using the `Route::livewire()` method.

```php
// routes/web.php
Route::livewire('test', 'cp.something');
```
Resources structure like:
```
resources\views\livewire\cp\something.blade.php
```

Not sure if its relevant but I'm using PHP 7.4 on **Windows 10**.

Is this a known / expected behaviour? Or am I doing something wrong?

**Comment by calebporzio** [comment]
Do you have a class under: `App\Http\Livewire\Cp\Something.php`?

Livewire uses `cp.something` to find the class of the component, then the `->render()` method is in charge of locating the view.

If you generate your Livewire components using `artisan make:livewire cp.something`, it will take care of all of this for you!

**Reply by johncarter-** [reply]
I did not, it was under `App\Http\Livewire\Something.php` returning `view('livewire.cp.something')`. 

Terribly sorry, old bean.
